### ISSUE TITLE
White screen when trying to stream, can move mouse on host

### ISSUE THREAD

**Issue by jazha** [moonlight-stream/moonlight-qt#200]
**Describe the bug**
White screen on stream start, can still move mouse on host

**Steps to reproduce**
Try streaming with Asus Zenbook UX430UNR, tested with Windows 10 Pro and Linux Mint 19

**Screenshots**
Screenshots will be only white, even in window mode, all white

**Affected games**
All games, including desktop streaming

**Moonlight settings (please complete the following information)**
- Have any settings been adjusted from defaults? I tried almost all possible combinations, software and hardware decoding. (only difference is black screen on software, and white screen on hardware) Tried different resolutions, and a bunch of other settings.
- If so, which settings have been changed? Almost all, then reverted to default
- Does the problem still occur after reverting settings back to default? Yes

**Client PC details (please complete the following information)**
 - OS: Windows 10 Pro 1809, (Build 17763.379) 
 - Moonlight Version: Tested 0.8.1, 0.9.0, 0.9.1 and 0.0.0.571 from appveyor

**Server PC details (please complete the following information)**
 - OS: Windows 10 Pro  1809, (Build 17763.379) (also tested with 1803, (Build 17134.648))
 - GeForce Experience version: 3.18.0.94
 - Nvidia GPU driver: 419.35

**Moonlight Logs (please attach)**
[Moonlight-1553384767.log](https://github.com/moonlight-stream/moonlight-qt/files/2999891/Moonlight-1553384767.log)

**Additional context**
I am unsure why it won't work with this laptop as the server can stream perfect to an Intel NUC I also use moonlight 0.9.1 on. All computers are connected within same network , same wifi and\or cabled ethernet connections

**Comment by cgutman** [comment]
Based on the fact that there's no rendering performance print at the end, it looks like no video packets are being received by your client PC.

Do you have any firewall software that might be blocking it on your client PC? If not, can you run Wireshark on your client PC and get a packet capture while you're trying to stream?

**Comment by jazha** [comment]
I have no firewall software on the client, except the standard windows firewall, issue also happens in linux.
The server does also not have anything else than the windows firewall.

I will doublecheck that it still works on the NUC, and I will run a wireshark on my laptop client and report back in a few.

**Comment by jazha** [comment]
I zipped the file as github does not support the .pcapng file ending, I filtered it to show only client and server ip traffic.
[wirefiltered.zip](https://github.com/moonlight-stream/moonlight-qt/files/3000235/wirefiltered.zip)
 
I am not too familiar with wireshark, so if I have done something wrong or you need more, please ask :)

**Comment by jazha** [comment]
One more detail, it works when I use an USB-C to ethernet dongle on the Asus UX430UNR laptop, issue only present when I use 5Ghz wifi on my laptop, 2.4Ghz seems fine aswell.

If it helps, my server is also running on the 5Ghz network.

**Comment by jazha** [comment]
After even more testing, it seems moving the host pc to cabled ethernet connection solved all the problems, I can now stream with 5Ghz on the laptop, and the issue is now a none-issue. It just will not work with host on wifi and client on wifi at the same time on 5Ghz network, every other combination works, although it lags on 2.4Ghz.

TL;DR:

Solved issue by moving server host to cabled ethernet and switching laptop back to 5Ghz wifi.

Will not work when both are on 5Ghz network.
